packages/fluentui/docs/src/examples/components/Card/Usage/CardExampleFocusable.tsx

```diff
@@ -0,0 +1,24 @@
+import { Flex, Image, Text, Card, cardFocusableBehavior } from '@fluentui/react-northstar';
+import * as React from 'react';
+
+const CardExampleFocusable = () => {
+  const [clickCount, setClickCount] = React.useState(0);
+  const updateClickCount = () => {
+    setClickCount(clickCount + 1);
```

Comment:
nit: 
```suggestion
    setClickCount(count => count + 1);
```

https://reactjs.org/docs/hooks-reference.html#functional-updates